component.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import style from 'app/components/fireworks/style.scss';
+
+export const FireworksComponent: React.FunctionComponent<{}> = React.memo((): JSX.Element => {
```

Comment:
Is this what we'll be displaying when a user successfully integrates?

Reply:
Yes